lib/capabilities.go

```diff
@@ -78,3 +91,31 @@ func CapAwayNotify(caps *CapManager) {
 		},
 	)
 }
+
+func CapServerTime(caps *CapManager) {
+	name := "server-time"
+	caps.Supported[name] = ""
+
+	caps.FnsInitListener[name] = func(listener *Listener) {
+		listener.TagsEnabled = true
+	}
+
+	caps.FnsMessageToClient = append(
+		caps.FnsMessageToClient,
+		func(listener *Listener, message *ircmsg.IrcMessage) bool {
+			if !listener.IsCapEnabled(name) {
+				return false
+			}
+
+			_, exists := message.Tags["time"]
+			if !exists {
+				message.Tags["time"] = ircmsg.TagValue{
```

Comment:
Hmm... I could see it go either way. Right now my view is that they mean different things, `name=` having a value that is an empty string where `name` has no value at all (i.e. is just a flag). I think bringing it up with IRCv3 WG makes sense to clarify how to handle it.